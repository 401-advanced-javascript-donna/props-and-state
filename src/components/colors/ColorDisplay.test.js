import React from 'react';
import Enzyme from 'enzyme';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });
describe('Color Display Component', () => {
  it('should render color', () => {
    const wrapper = shallow(<ColorDisplay backgroundColor='red'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
