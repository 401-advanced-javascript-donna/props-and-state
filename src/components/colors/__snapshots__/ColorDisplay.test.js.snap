// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Display Component should render color 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
